LogicalTypes.md

```diff
@@ -684,44 +702,67 @@ optional group my_list (LIST) {
 }
 ```
 
-Some existing data does not include the inner element layer. For
-backward-compatibility, the type of elements in `LIST`-annotated structures
-should always be determined by the following rules:
+##### 2-level structure
+
+Some existing data does not include the inner element layer, resulting in a
+`LIST` that annotates a 2-level structure. Unlike the 3-level structure, the
+repetition of a 2-level structure can be `optional`, `required`, or `repeated`.
+When it is `repeated`, the `LIST`-annotated 2-level structure can only serve as
+an element within another `LIST`-annotated 2-level structure.
+
+```
+<list-repetition> group <name> (LIST) {
+  repeated <element-type> <element-name>;
+}
```

Comment:
Again, I think that calling attention to the degenerate cases and documenting them is only going to cause more confusion. The purpose of this originally was to simply document how to interpret data that doesn't match expectations. Now this introduces how a 2-level list looks, which I think increases the possibility that people will misread this and write them.

Reply:
Removed